testing/trino-testing/src/main/java/io/trino/testing/AbstractTestQueryFramework.java

```diff
@@ -190,7 +193,9 @@ public void ensureTestNamingConvention()
 
     protected Session getSession()
     {
-        return queryRunner.getDefaultSession();
+        return Session.builder(queryRunner.getDefaultSession())
+                .setQueryId(queryIdGenerator.createNextQueryId())
```

Comment:
`getSession()` is an accessor, it should return same state on each invocatin

also, query id generation isn't this class's concern; it's a detail of the query runner